### PR TITLE
Mutations testing

### LINKED ISSUE #33: Make tests for create mutations

### PR DESCRIPTION
**Pull request by markcyen**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes #33 

### Current behavior
Description: No testing for creating a user, a dog, or an event.

### New behavior
Description: Mutations test for creating user, dog, and event. Brought test coverage to 90.91% 😄

## Type of change
 - Type of change is just added testing to mutations.
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
# Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by kon-ham** [comment]
Looks like we have a failing Travis CI test - would you like for me to approve?

**Comment by markcyen** [comment]
> Looks like we have a failing Travis CI test - would you like for me to approve?

Any ideas on how to get past the eventDatetime? ...remembering you telling me that Travis CI is on EDT?

**Comment by kon-ham** [comment]
Mark, sorry about the `event` thing. It should be `dog: nill` or whatever.

No idea how to make it pass the dateTime thing. I think we could skip that and see what Brian thinks later.

**Comment by markcyen** [comment]
> Mark, sorry about the `event` thing. It should be `dog: nill` or whatever.
> 
> No idea how to make it pass the dateTime thing. I think we could skip that and see what Brian thinks later.

maybe there is a way to format the datetime in the `event_type.rb` file...

**Comment by markcyen** [comment]
> > Mark, sorry about the `event` thing. It should be `dog: nill` or whatever.
> > No idea how to make it pass the dateTime thing. I think we could skip that and see what Brian thinks later.
> 
> maybe there is a way to format the datetime in the `event_type.rb` file...

this shall pass...

**Review by kon-ham — APPROVED** [review]

**Comment by kon-ham** [comment]
Nice fix - you decided to just include month and day? Awesome.

File: spec/graphql/mutations/events/create_event_spec.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          json = JSON.parse(response.body, symbolize_names: true)
+          data = json[:data][:createEvent][:event]
+
+          expect(data).to include(
+            id: Event.last.id.to_s,
+            dogId: @dog.id,
+            name: 'Potty training',
+            completed: false
+          )
+          expect(data[:eventDatetime][0..9]).to eq("2021-08-27")
```
Comment (EmmyMorris) [inline spec/graphql/mutations/events/create_event_spec.rb:29]: Great work around!!

**Review by EmmyMorris — COMMENTED** [review]
This looks great! I'm glad changing the format seemed to help!

**Pull request merged**